aten/src/ATen/native/vulkan/ops/Factory.cpp

```diff
@@ -55,7 +56,12 @@ Tensor empty_strided(
     const optional<Device> device,
     const optional<bool> pin_memory) {
   return empty_memory_format(
-      sizes, dtype, layout, device, pin_memory, c10::MemoryFormat::Contiguous);
+      c10::SymIntArrayRef::fromIntArrayRef(sizes),
```

Comment:
These conversions are not cost free, but fortunately in the end state you'll only do a representability check on entry to dispatcher, and then a "not symint" test once you exit all the composite nonsense.